### PR TITLE
New version: hikari-no-yume.touchHLE version 0.2.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/118643)

### CODE REVIEW

**Comment by stephengillie** [comment]
2:22:59 PM: 
> 2023-09-01 14:22:58.118 [CLI ] Package hash verification failed. SHA256 in manifest [a6a8c11ad53b851c321027cde651ac8ad575505234ef6c614ad4e28eb3fe219c] does not match download [c170c67d47fb9296ec740b8049ae89085b64d5748b3cd624b4bee69733817cd4]

File: manifests/h/hikari-no-yume/touchHLE/0.2.0/hikari-no-yume.touchHLE.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+PackageVersion: 0.2.0
+Installers:
+- Architecture: x64
+  InstallerType: zip
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: touchHLE.exe
+    PortableCommandAlias: touchHLE
+  InstallerUrl: https://github.com/hikari-no-yume/touchHLE/releases/download/v0.2.0/touchHLE_v0.2.0_Windows_x86_64.zip
+  InstallerSha256: A6A8C11AD53B851C321027CDE651AC8AD575505234EF6C614AD4E28EB3FE219C
```
Comment (stephengillie) [inline manifests/h/hikari-no-yume/touchHLE/0.2.0/hikari-no-yume.touchHLE.installer.yaml:14]:
```suggestion
  InstallerSha256: C170C67D47FB9296EC740B8049AE89085B64D5748B3CD624B4BEE69733817CD4
```

**Comment by stephengillie** [comment]
image

**Comment by Trenly** [comment]
[Policy] Unblocked

File: manifests/h/hikari-no-yume/touchHLE/0.2.0/hikari-no-yume.touchHLE.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: hikari-no-yume.touchHLE
+PackageVersion: 0.2.0
```
Comment (stephengillie) [inline manifests/h/hikari-no-yume/touchHLE/0.2.0/hikari-no-yume.touchHLE.installer.yaml:5]:
```suggestion
PackageVersion: 0.2.0
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Comment by stephengillie** [comment]
image

* https://github.com/microsoft/winget-pkgs/issues/107639

**Review by stephengillie — APPROVED** [review]

**Pull request merged**